ecs-init/version/version.go

```diff
@@ -18,7 +18,7 @@
 package version
 
 // Version is the version of the ecs-init
-var Version string = "unknown"
+var Version string = "1.15.2"
```

Comment:
Since this is set at build time similarly to the `GitShortHash`, I think we should leave it as `unknown` also.